data-structures/linked-lists.md

```diff
@@ -149,7 +144,7 @@ ___
 
 *Make sure to mention these things:*
 - Common data structures in interviews (hash tables, binary search trees, etc.)
-- Most blockchains are built on some implementation of the merkle tree data structure patented by Ralph Merkle (check out his site -> merkle.com for more info if you're into cryptograpghy)
+- Most blockchains are built on some implementation of the Merkle tree data structure patented by Ralph Merkle (check out his site -> merkle.com for more info if you're into cryptography)
```

Comment:
Add the hyperlink too so that the user don't have to manually copy paste the link.